src/strategies/terms_stats.js

```diff
@@ -0,0 +1,34 @@
+import _ from 'lodash/fp'
+import { runWith } from '../utils'
+
+export default async ({ service, tree, ...node }) => {
+  let { key_field, size = 100 } = node
+  let run = node => runWith(service, tree, node)
+
+  let totalRecords = _.get(
+    'context.value',
+    await run({
+      key: 'cardinality',
+      type: 'cardinality',
+      field: key_field,
+      fieldMode: 'autocomplete',
+    })
+  )
+
+  let terms_stats = {
+    getTotalRecords: () => {
+      return totalRecords
+    },
+    async *[Symbol.asyncIterator]() {
+      let node = await run({
```

Comment:
same here - I just noticed you're not writing `terms_stats.node` which we started doing in this version to make debugging easier